Parser/readWQMraw.m

```diff
@@ -1,542 +1,568 @@
-function sample_data = readWQMraw( filename, mode )
-%readWQMraw parses a .RAW file retrieved from a Wetlabs WQM instrument.
-%
-% This function is able to parse raw data retrieved from a Wetlabs WQM CTD/ECO
-% instrument.
-%
-% Inputs:
-%   filename    - name of the input file to be parsed
-%   mode        - Toolbox data type mode.
-%
-% Outputs:
-%   sample_data - contains a time vector (in matlab numeric format), and a
-%                 vector of up to nine variable structs, containing sample
-%                 data. The possible variables are as follows:
-%
-%                   Conductivity      ('CNDC'): S m^-1
-%                   Temperature       ('TEMP'): Degrees Celsius
-%                   Pressure          ('PRES_REL'): Decibars
-%                   Salinity          ('PSAL'): 1e^(-3) (PSS)
-%                   Dissolved Oxygen  ('DOXY'): mg/l
-%                   Dissolved Oxygen  ('DOX1'): mmol/m^3
-%                   Dissolved Oxygen  ('DOX'): ml/l
-%                   fluorescence      ('CPHL'): mg/m^3
-%                   Chlorophyll       ('CHLU'): mg/m^3   (user coefficient)
-%                   Chlorophyll       ('CHLF'): mg/m^3   (factory coefficient)
-%                   Turbidity         ('TURB') NTU
-%
-%                 Also contains some metadata fields and calibration information, so
-%                 the following are present:
-%
-%                   meta.instrument_make:      'WET Labs'
-%                   meta.instrument_model:     'WQM'
-%                   meta.instrument_serial_no: retrieved from input file
-%                   meta.instrument_sample_interval: retrieved from input file
-%                   Calibration: retrieved from input file
-%
-% Author:       Charles James <charles.james@sa.gov.au>
-% Contributor:  Guillaume Galibert <guillaume.galibert@utas.edu.au>
-%
-
-%
-% Copyright (C) 2017, Australian Ocean Data Network (AODN) and Integrated
-% Marine Observing System (IMOS).
-%
-% This program is free software: you can redistribute it and/or modify
-% it under the terms of the GNU General Public License as published by
-% the Free Software Foundation version 3 of the License.
-%
-% This program is distributed in the hope that it will be useful,
-% but WITHOUT ANY WARRANTY; without even the implied warranty of
-% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-% GNU General Public License for more details.
-
-% You should have received a copy of the GNU General Public License
-% along with this program.
-% If not, see <https://www.gnu.org/licenses/gpl-3.0.en.html>.
-%
-% ensure that there are exactly two arguments
-narginchk(2, 2);
-if ~ischar(filename), error('filename must contain a string'); end
-params = load_params();
-wqmdata = load_wqm(filename);
-sample_data = load_sample_data(filename,mode,params,wqmdata);
+function sample_data = readWQMraw(filename, mode)
+    %readWQMraw parses a .RAW file retrieved from a Wetlabs WQM instrument.

```

Comment:
Again, impossible to review if you introduce a change on the whole file.

Reply:
again, no need for review, the commit is just for that, indentation. PS: this is automatically done, I'm just splitting in commits so its not mangled with proper changes.